.travis.yml

```diff
@@ -4,7 +4,6 @@ go:
     - tip
     - 1.5.1
     - 1.5
-    - 1.4.3
```

Comment:
Good point. I think the latest is enough, but it's easier. Also, it's the most secure way to support all versions.